webapp/packages/core-blocks/src/FormControls/InputField/InputFieldBase.tsx

```diff
@@ -116,9 +117,10 @@ export const InputFieldBase = observer<InputFieldBaseProps, HTMLInputElement>(
           <input
             ref={mergedRef}
             {...rest}
-            type={passwordRevealed ? 'text' : rest.type}
+            type={type}
             name={name}
             value={uncontrolled ? undefined : value}
+            autoComplete={type === 'password' ? 'new-password' : rest.autoComplete}
```

Comment:
seems like unwanted change, because for password we want default browser behaviour, we don't want to treat any password field as new-password, please revert